### ISSUE TITLE
Python 3 print error

### ISSUE THREAD

**Issue by jklemm** [atodorov/django-s3-cache#4]
Hi @atodorov,

I want to migrate a legacy Django 1.8 software to Python 3, and upgrading the `django-s3-cache` to latest (1.4.2), I got this error:

```
Collecting django-s3-cache==1.4.2 (from -r requirements.txt (line 16))
  Downloading https://files.pythonhosted.org/packages/82/b9/bca55ce21c9f8a83684985e40d2b7c5aa8e4fde4d9bc9ecf21b9c5c7441a/django-s3-cache-1.4.2.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-rcu0tsqf/django-s3-cache/setup.py", line 73
        print req
                ^
    SyntaxError: Missing parentheses in call to 'print'. Did you mean print(req)?
------
 Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-rcu0tsqf/django-s3-cache/
```

Looking at master branch, the issue is solved (using print as a function).

Do you plan launch a new release with master branch's code?

Cheers! :brazil:

**Comment by atodorov** [comment]
@jklemm feel free to send a pull request and I can cut a new release afterwards if you need it.

**Comment by jklemm** [comment]
Looking at master branch, the issue is solved (using print as a function).

Can you cut a new release now with master branch's code?

**Comment by jklemm** [comment]
Hi @atodorov 

We have solved the problem dropping `django-s3-cache==1.3` and upgrading `django-storages` from `1.1.8` to `1.6.5` (this version supports Django 1.8).

This is only possible because we're using Redis to cache, and don't use s3 anymore.

Other required change was an import:
- From `from storages.backends.s3boto import S3BotoStorage`
- To `from storages.backends.s3boto3 import S3Boto3Storage`

This class is found on `boto3` package.

Your library was extremely used for years.

Thanks a lot.
